mks_server.py

```diff
@@ -212,12 +226,19 @@ def update_readings(self):
                         data2 = 0
                     else:
                         data2 = float(rdg2)
-                except ValueError:
+                except ValueError as e:
+                    logging.debug('Serial reading 2 error: {}'.format(e))
+                    error = True
                     data2 = float('NaN')
+                if error:
+                    logging.debug('Pressure: {}'.format(result.pressure))
+                    logging.debug('Full Scale: {}'.format(result.full_scale))
```

Comment:
I'd add a log message saying that there was, in fact, an error. Otherwise it's not entirely obvious why these values are being logged, particularly since they'll be messed up by construction.